crates/oxvg_optimiser/src/utils/minify_style.rs

```diff
@@ -7,14 +7,14 @@ use lightningcss::{
     stylesheet::{MinifyOptions, ParserOptions, StyleAttribute, StyleSheet},
 };
 
-pub(crate) fn style<'i>(style: &mut DeclarationBlock<'i>) {
+pub(crate) fn style(style: &mut DeclarationBlock<'_>) {
```

Comment:
```suggestion
pub(crate) fn style(style: &mut DeclarationBlock) {
```
I think elided lifetimes like this can be removed completely

Reply:
updated, it's seems not reported by clippy as confusing elided lifetime